src/pages/students/index.tsx

```diff
@@ -16,19 +16,19 @@ const Students = () => {
     onError: (errortraits) => {
       // errortraits allows one to access validation, code, message, and path
       // JSON.parse is utilized because errortraits.message is a string
-      const formattederrortraits = JSON.parse(errortraits.message) as {
-        validation: string;
-        code: string;
-        message: string;
-        path: string[];
-      }[];
+      try {
+        const formattederrortraits = JSON.parse(errortraits.message) as {
```

Comment:
Would recommend renaming "errortraits" to "e", "err", or "error" to fit more with standard convention, would similarly recommend renaming "formattederrortraits" based on that and using camelCase for the new name or this one if you keep it.